src/bin/juliainstaller.rs

```diff
@@ -234,17 +256,23 @@ pub fn main() -> Result<()> {
     }
 
     let mut install_choices = InstallChoices {
-        backgroundselfupdate: 0,
-        startupselfupdate: 1440,
+        backgroundselfupdate: args.background_selfupdate_interval.parse().unwrap(),
+        startupselfupdate: args.startup_selfupdate_interval.parse().unwrap(),
```

Comment:
In general we should not use `unwrap` for anything where a user might input incorrect values, the error message is not pretty for it ;) I think in this case the right strategy would be to use a numeric data type in `clap`, and then we don't even have to worry about this.

Reply:
See my resolved comments above -- I've cleaned up the parser.